commcare_connect/organization/views.py

```diff
@@ -25,10 +28,19 @@ def organization_home(request, org_slug):
     if not form:
         form = OrganizationChangeForm(instance=org)
 
+    credentials = connect_id_client.fetch_credentials()
+    if not any(c.name == f"Worked for {org.name}" for c in credentials):
+        credentials.append(Credential(name=f"Worked for {org.name}", slug="default"))
```

Comment:
what does it mean that slug is always default here?

Reply:
This is here in case there are no credentials present in the database. If a credential with the same name is present the default credential is not added to the list.

Reply:
Is a credential added to the DB with a slug `default` ever?

Reply:
Nope, we only send the name of the credential that is to be added. The slugs are generated in ConnectID from the credential names.